changelog/fix_speed_up_single_file_inspection.md

```diff
@@ -0,0 +1 @@
+* [#10889](https://github.com/rubocop/rubocop/pull/10889): Skip forking off extra processes for parallel inspection when only a single file needs to be inspected. ([@wjwh][])
```

Comment:
```suggestion
* [#1003](https://github.com/rubocop/rubocop/pull/10903): Skip forking off extra processes for parallel inspection when only a single file needs to be inspected. ([@wjwh][])
```

Reply:
Whoops, leftover from the original PR where something went wrong with the rebasing. Thanks for catching that.